docs/Queries.md

```diff
@@ -90,8 +121,14 @@ A query response is also a plain object
   ref: string,
   value: any,
   type?: string,
-  flags?: array,
-  meta?: object,
+  meta?: {
+    flags?: string[],
+    variants?: {
+      [string]: {  // experiment
+        [string]: string  // context: variant
```

Comment:
I'd be interested in your recommendation on this - chapterId and memberId seem like they should be pretty safe as integers, but yeah I'm not sure if that is a hard rule for what 'context' can be in the variants service... maybe I can just declare `[string|number]` as the type and let consumers determine what they should use.